### DISCUSSION TITLE
Battery Health and Battery Health - LFP degradation difference

### DISCUSSION THREAD

**Discussion by c040656** [teslamate-org/teslamate#3621] [Q&A]
Hi everyone,

My Tesla Model Y RWD LFP is now 4 months old. Afew days go I tried to calibrate the battery by charging to 100% and Usable (now) is reaching 60 kWh now. However I start to realized that degradation showing under Battery Health dashboard is different to Battery Health - LFP dashboard.

I have checked a few green dots on the "Battery Health by Mileages" under these two dashboards. The kWh per green dots under Battery Health - LFP dashboard are always 2 kWh less than Battery Health dashboard.

Which degradation should I trust? And how are kWh number under each green dot coming from?

Thanks in Advance.

![Screenshot 2024-01-18 at 10 10 23 PM](https://github.com/teslamate-org/teslamate/assets/91834487/b34152b6-370b-488a-ba05-871bfc1a6775)
![Screenshot 2024-01-18 at 10 10 29 PM](https://github.com/teslamate-org/teslamate/assets/91834487/93db10ec-fe79-49e2-95b5-2e4f87915f80)

**Comment by cwanja** [comment]
@jheredianet any details?

**Reply by jheredianet** [reply]
Hi, I can only speak for the original Battery "Health dashboard" which was the one I've created. After releasing it on Teslamate somebody made the changes for LFP, but I'm not sure what has changed (I've just made a quick review), in my humble opinion I think there is no need to have 2 dashboards (since the data in Teslamate is already based on the type of battery you have) as commented here https://github.com/teslamate-org/teslamate/discussions/3513

Maybe this could help to understand the difference between these dashboards 
https://github.com/teslamate-org/teslamate/pull/3311

Actually, I've made a PR with some improvements for the original dashboard here 
https://github.com/teslamate-org/teslamate/pull/3593

If you wish to test the last version of this dashboard (and other ones) you could check this repository https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards
